services/mailer.js

```diff
@@ -0,0 +1,23 @@
+const mailgun = require('mailgun-js');
+const logger = require('../lib/log')
+require('dotenv').config()
+
+const mg = mailgun({apiKey: process.env.MAILGUN_API_KEY, domain: process.env.DOMAIN});
+
+const sendConfirmationEmail = () => {
+    const data = {
+        from: process.env.SENDER_EMAIL,
+        to: process.env.RECEIVER_EMAIL,
+        subject: 'Congratulations!',
+        text: 'Welcome to C0D3',
+        html: "<h1>Confirm your E-mail</h1><button>Confirm</button>"
+    };
+    mg.messages().send(data, function(error, body){
```

Comment:
there's no way for caller to see whether email was successful or not. You should return a promise that resolves or rejects. From the documentation, it seems like the library returns promises: https://www.npmjs.com/package/mailgun-js#promises